docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -243,33 +246,34 @@ The next step in testing your new module is to consume it with an Ansible playbo
 
 - Run the playbook and analyze the output: ``$ ansible-playbook ./testmod.yml``
 
-Testing basics
-====================
+Testing your newly-created module
+=================================
 
-These two examples will get you started with testing your module code. Please review our :ref:`testing <developing_testing>` section for more detailed
+The following two examples will get you started with testing your module code. Please review our :ref:`testing <developing_testing>` section for more detailed
 information, including instructions for :ref:`testing module documentation <testing_module_documentation>`, adding :ref:`integration tests <testing_integration>`, and more.
 
-Sanity tests
-------------
+Performing sanity tests
+-----------------------
 
 You can run through Ansible's sanity checks in a container:
 
 ``$ ansible-test sanity -v --docker --python 2.7 MODULE_NAME``
 
-Note that this example requires Docker to be installed and running. If you'd rather not use a
-container for this, you can choose to use ``--venv`` instead of ``--docker``.
+.. note:: 
+	Note that this example requires Docker to be installed and running. If you'd rather not use a container for this, you can choose to use ``--venv`` instead of ``--docker``.
 
-Unit tests
-----------
+Performing unit tests
```

Comment:
The sanity tests already exist, but for a new module, the developer must add new unit tests.
```suggestion
Adding unit tests
```